biochemist-python/chapters/introduction.ipynb

```diff
@@ -85,23 +85,12 @@
   },
```

Comment:
<p>Is there a specific reason to change this? I like <code>7 + 3</code> because  it's something the audience will know the answer to (they're able to verify that Python <em>does</em> work like a calculator). Just wondering if there's a specific reason to change it to multiplication.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MolSSI-Education/python-scripting-biochemistry/pull/20/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>